cardInfoAPI-Level1.yaml

```diff
@@ -0,0 +1,1402 @@
+openapi: 3.1.0
```

Comment:
Some tools might still be catching up with "openapi: 3.1.0", which could lead to compatibility issues. Is it required to move to "openapi: 3.1.0" already

Reply:
OpenAPI 3.1 is planned to be the standard version for all Common API standards. With the Card API we already want to start with this version.
We expect that the tool support will be improved until the Card API standard is adopted.